Aggiunge capitolo Cosa significa essere dev (#111)
Il-Libro-Open-Source/book: docs/it/cosa-significa-essere-dev.md

_Cosa significa "essere dev"?_

La domanda, di per sé, è piuttosto semplice.

Non si può dire però altrettanto per la risposta... o, per meglio dire, le risposte.

> Ebbene sì, essere dev va oltre una singola e banale definizione: a tale domanda si potrebbe rispondere "_Essere dev significa ricoprire una figura professionale dedita alla stesura del codice e con il fine ultimo di creare prodotti digitali_"... ma sarebbe riduttivo.

Reviewer: ```suggestion
Ebbene sì, essere dev va oltre una singola e banale definizione: a tale domanda si potrebbe rispondere "_Essere dev significa svolgere una professione dedita alla stesura del codice e con il fine ultimo di creare prodotti digitali_"... ma sarebbe riduttivo.
```

Caratteristiche che nulla hanno a che fare con la scrittura del codice, né presentano alcun tipo di legame diretto con specifiche competenze tecniche; nonostante ciò, risultano essere caratteristiche **fondamentali** per intraprendere questa carriera al meglio, lasciarsi coinvolgere da essa e riuscire a portare valore in questo settore.

## La passione: il combustibile per eccellenza

Decidere di "diventare dev" significa scegliere di "sacrificare" parte del proprio tempo al fine di raggiungere un obiettivo; non sarà un percorso facile, tantomeno breve; ci saranno momenti di eccitazione, ma anche di sconforto; ci saranno giorni in cui tutto sembrerà andare per il verso giusto, ed altri in cui ogni nostra singola attività verrà minata dalle difficoltà che ci si pareranno davanti; ci saranno concetti che domeremo con estrema facilità, ed altri in cui ne usciremo sconfitti e con la coda tra le gambe.

> Fa tutto parte del gioco: _abbraccialo, comprendilo e fallo tuo_; magari non oggi e forse nemmeno domani, ma arriverà il momento in cui riuscirai dove prima hai fallito.

Reviewer: ```suggestion
Fa tutto parte del gioco: _abbraccialo, comprendilo, e fallo tuo_; magari non oggi e forse nemmeno domani, ma arriverà il momento in cui riuscirai dove prima hai fallito.
```

Proprio come questo libro, in rete esistono miliardi di informazioni a cui puoi avere accesso: aspettano solo di essere ricercate! Queste pagine che stai leggendo sono presenti sul web, e molto probabilmente ne sei a conoscenza perché è stato semplice imbattersi in esse navigando per la rete, o magari perché chi ti ha consigliato questo libro lo ha cercato a sua volta. Poco importa, quello che è davvero importante è che tu prenda consapevolezza del fatto che il web pullula di informazioni a cui puoi accedere, e che l'unica cosa che ti separa da esse, indovina un po', _sei tu_!

Le informazioni presenti in rete però, per quanto siano molteplici e liberamente fruibili, da sole non hanno alcun effetto sulla tua formazione; analogamente a quanto avviene con un esplosivo, la polvere da sparo, per quanto potente possa essere, da sola non basta; serve un innesco o, per meglio dire, serve **una scintilla che sia in grado di innescare la miccia**.

A questo punto, a te che leggi, avrai capito dove vogliamo andare a parare: ciò che non deve mai mancare in te è la curiosità!

> Essa sarà la scintilla in grado di innescare in te una forma mentis vincente, prona all'apprendimento e alla crescita professionale; fintanto che questa scintilla resterà accesa, non avrai limiti nell'apprendimento: anche qui, l'unico limite che avrai, _sarai tu_.

Reviewer: Riformulerei, la scintilla non resta accesa per molto

```suggestion
Essa sarà la scintilla in grado di innescare in te una forma mentis vincente, prona all'apprendimento e alla crescita professionale; fintanto che questa scintilla continuerà a scattare, non avrai limiti nell'apprendimento: anche qui, l'unico limite che avrai, _sarai tu_.
```
